docs/content/latest/api/ysql/grammar_diagrams.md

```diff
@@ -154,9 +154,9 @@ cache = '<Integer Literal>';
 
 ### create_table
 ```
-create_table = 'CREATE' 'TABLE' [ 'IF' 'NOT' 'EXISTS' ] table_name '(' [ table_elem { ',' table_elem } ] ')';
+create_table = 'CREATE' 'TABLE' [ 'IF' 'NOT' 'EXISTS' ] table_name '(' [ table_elem { ',' table_elem } ] ')' [ 'WITH' '(' storage_param { ',' storage_param } ')' | 'WITHOUT' 'OIDS' ];
```

Comment:
This grammar should now be generated differently by the diagram generator (but we haven't replaced all entries yet).